ui/js/components/perf/compare.js

```diff
@@ -108,3 +109,33 @@ treeherder.component('compareError', {
         newRevision: '<'
     }
 });
+
+treeherder.component('distributionGraph', {
+    template:`
+        <table class = "toolTipTable"> 
```

Comment:
should be no spaces between `class`, `=` and the value (same for everything else).